source/user-manual/agents/remote-upgrading/create-custom-wpk/generate-wpk-package-manually.rst

```diff
@@ -131,6 +131,69 @@ Compile the WPK package using the MSI package and, your SSL certificate and key:
 
   # tools/agent-upgrade/wpkpack.py output/myagent.wpk path/to/wpkcert.pem path/to/wpkcert.key path/to/wazuhagent.msi path/to/upgrade.bat path/to/do_upgrade.ps1
 
+MacOS WPK
+^^^^^^^^^
+
+Install development tools and compilers. In macOS, this can be easily done by installing brew, a package manager for macOS:
```

Comment:
This is not OK. MacOS uses `.pkg` file, no compiled files.

Reply:
Done